js/index.js

```diff
@@ -1,6 +1,27 @@
 // Iteration 1: Names and Input
+const hacker1 = "Marten";
+const hacker2 = "Bart";
 
-// Iteration 2: Conditionals
+console.log("The driver's name is " +hacker1);
+console.log("The navigator's name is " +hacker2);
+
+
+// Iteration 2: Conditionals and Iteration 3: Loops
+if (hacker1.length > hacker2.length){
+    console.log("The driver has the longest name, it has " + hacker1.length + " characters.");
+    console.log("the drivers name goes first")
+  } else if (hacker2.length > hacker1.length) {
+    console.log("It seems that the navigator has the longest name, it has " + hacker2.length + " characters.");
+    console.log("Yo, the navigator goes first definitely.");
+  } else {
+    console.log("Wow, you both have equally long names, " + hacker1.length + " characters!");
+    console.log("What?! You both have the same name?");
+  }
+   
+  var resultsDriver = hacker1
+  console.log(resultsDriver.split('').join(' ').toUpperCase());
+  
```

Comment:
var is not really used nowadays. And you could have just written hacker1 instead of resultsDriver
or resultsDriver = hacker1.split('').join(' ').toUpperCase()

This is a great solution for later! But actually you should use a for loop here